### PR TITLE
Getting /newGift route working withAuth()

### PR DESCRIPTION
**Pull request by Donnastjames**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const hbs = exphbs.create({ helpers });
 const sess = {
   secret: 'Super secret secret',
   cookie: {
-    maxAge: 3600,
+    maxAge: 60 * 60 * 1000, // One hour!
     httpOnly: true,
     secure: false,
     sameSite: 'strict',
```
Comment (Donnastjames) [inline server.js:23]: I only fixed a bit of whitespace in auth.js, and for some reason GitHub has troubles showing it below.

File: controllers/homeRoutes.js [hunk 2]
```diff
@@ -69,4 +73,34 @@ router.get('/gifts/:id', async (req, res) => {
   }
 });
 
+// Use withAuth middleware to prevent access to route
```
Comment (Donnastjames) [inline controllers/homeRoutes.js:76]: Cailin, I forgot to comment out this code.  Sorry!  just go ahead and comment it out prior to merging after the approvals.  Thanks!

**Review by CailinBellWold — APPROVED** [review]
Walked through this with Donna in group meeting. Looks good.

**Review by DannyRoubin — APPROVED** [review]
Great work Donna! Looks great

**Review by tylerhance — APPROVED** [review]
Looks good, great work!

**Pull request merged**